packages/sdk/src/cosmwasmclient.spec.ts

```diff
@@ -187,6 +183,7 @@ describe("CosmWasmClient", () => {
         };
 
         const result = await client.postTx(marshalTx(signedTx));
+        await sleep(50); // wait until tx is indexed
```

Comment:
I really wish there was a way to do without sleep, but I didn't find it either with weave